crates/input/src/io.rs

```diff
@@ -0,0 +1,39 @@
+use std::io::Read;
+
+use de_core::fs::conf_dir;
+use leafwing_input_manager::prelude::InputMap;
+use ron::ser::PrettyConfig;
+
+use crate::BindableActionlike;
+
+pub(crate) fn get_keybindings<A: BindableActionlike>(
```

Comment:
I would not introduce any (de)serialization in this PR. Lets first migrate to leafwing_input_manager to keep the PR lean.